src/components/LabelSection/LabelSection.jsx

```diff
@@ -1,10 +1,125 @@
-import React from 'react';
+import React, { useState } from "react";
+import { useNotes } from "context";
 import "./LabelSection.css";
 
-const LabelSection = () => {
+const LabelSection = ({ activeDropdown, setActiveDropdown }) => {
+    const { noteLabels, setNoteLabels, noteData, dispatchNoteData } = useNotes();
+    const toggleLabelDropdown = () => {
+        setActiveDropdown((currentValue) => ({
+            priorityDropdown: false,
+            colorDropdown: false,
+            labelDropdown: !currentValue.labelDropdown,
+        }));
+    };
+
+    const setNoteLabelsOnButtonClick = (e) => {
+        if (e.keyCode === 13) {
+            e.preventDefault();
+            if (labelInput.trim() !== "") {
+                setNoteLabels((prevList) =>
+                    prevList.includes(labelInput)
+                        ? [...prevList]
```

Comment:
what is the use of keynode here?

Reply:
To make sure that the action of adding a tag happens when we press enter key